src/components/tooltip/tooltip.hbs

```diff
@@ -2,15 +2,13 @@
   Tooltip label
   <div tabindex="0" data-tooltip-trigger data-tooltip-target="#unique-tooltip" role="tooltip" class="bx--tooltip__trigger">
     <svg width="16" height="16" viewBox="0 0 16 16">
-      <g fill-rule="evenodd">
-        <path d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z" fill-rule="nonzero" />
-        <path d="M8.5 7.007v3.991h2v1h-2V12h-1v-.002h-2v-1h2V7.007h-.995v-1h2v1H8.5z" fill-rule="nonzero" />
-        <circle cx="8" cy="4" r="1" />
-      </g>
+      <path d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z" fill-rule="nonzero" />
+      <path d="M8.5 7.007v3.991h2v1h-2V12h-1v-.002h-2v-1h2V7.007h-.995v-1h2v1H8.5z" fill-rule="nonzero" />
+      <circle cx="8" cy="4" r="1" />
     </svg>
   </div>
 </div>
-<div id="unique-tooltip" data-floating-menu-direction="bottom" class="bx--tooltip">
+<div id="unique-tooltip" data-floating-menu-direction="top" class="bx--tooltip">
```

Comment:
Is this change here intentional…?

Reply:
Nope, I was just testing that both top and bottom worked correctly. I'll swap it back.